tests/test_esutils.py

```diff
@@ -51,3 +51,57 @@ def test_is_get_query_with_results_valid_response_returns_true(self):
         es_get_response = {EsDocProperties.TYPE: "type", EsDocProperties.INDEX: "index", EsConst.FIELDS: {},
                            EsDocProperties.VERSION: 1, EsConst.FOUND: True, EsDocProperties.ID: "id"}
         self.assertTrue(EsUtils.is_get_query_with_results(es_get_response))
+
+    def test_has_aggregation_results_true(self):
```

Comment:
add test that return a valid result for agg_name, but we search for other name that we didn't request